.github/workflows/golangci-test.yml

```
@@ -0,0 +1,24 @@
+name: golangci-test
+on:
+  push:
+    tags:
+      - v*
+    branches:
+      - develop
+  pull_request:
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '1.14'
```

Please bump this to `1.16` which is being used in other action workflows, sometime later, we'd need to change it to `1.17` as in all workflows & go.mod file.
```suggestion
          go-version: '1.16'
```